unit/Makefile

```diff
@@ -67,4 +67,4 @@ platypus_unit_deps := $(patsubst %.C, %.$(obj-suffix).d, $(platypus_unit_srcfile
 -include $(platypus_unit_deps)
```

Comment:
For adding the remaining unit tests - MOOSE has a few examples with some external files in its unit tests in the `unit/data` directory - should work similarly here, which should increase the low test coverage arising (in part) due to the lost hephaestus tests